napari/components/viewer_model.py

```diff
@@ -503,7 +496,7 @@ def add_volume(
             {'opaque', 'translucent', and 'additive'}.
         visible : bool
             Whether the layer visual is currently being displayed.
-        spacing : list, optional
+        scale : list, optional
             List of anisotropy factors to scale the volume by. Must be one for
             each dimension.
```

Comment:
"Must be one for each dimension" -> "The length should be equal to the number of dimensions". Otherwise it sounds like it must literally be [1, 1, ...].

Reply:
fixed